api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -560,6 +565,22 @@ def delete_venue(venue_id: int) -> utils.BackofficeResponse:
     return redirect(url_for("backoffice_web.pro.search_pro"), code=303)
 
 
+@venue_blueprint.route("/<int:venue_id>/fully-sync", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.MANAGE_PRO_ENTITY)
+def fully_sync_venue(venue_id: int) -> utils.BackofficeResponse:
+    venue = get_venue(venue_id)
+    has_active_provider = any(venue_provider.isActive for venue_provider in venue.venueProviders)
+
+    if not has_active_provider:
+        flash(f"Impossible de lancer la synchronisation du lieu {venue.common_name} ({venue.id})", "warning")
+        return redirect(url_for("backoffice_web.venue.get", venue_id=venue_id), code=303)
+
+    fully_sync_venue_job.delay(venue_id)
```

Comment:
La résynchronisation totale d'un lieux ne peux se faire que dans certains cas (je ne suis pas sur de tous les connaitre). On a plusieurs types d'api, certaines où nous exposons des endpoints et les partenaires nous appellent, une autre où nous allons appeler les serveurs de nos partenaires. La resynchronisation ne peut (et doit) uniquement être lancé sur les 2emes, si on lance fully_resync_venue_job sur les 1er cas on risque de reset tout les stocks (1ere action de fully_sync_venue) sans pouvoir synchroniser ensuite. 
Les providers possiblement resynchronisables sont : 
-  Ceux dont la colonne apiUrl est not null
-  Ceux dont la colonne localClass est not null (je suis pas sur, chaque synchro a du code spécifique et je ne sais pa si elles supportent toutes le resync)

Reply:
Peut-être qu'il faut prendre le problème dans l'autre sens : sur quels providers on peut resynchroniser totalement ? 
Titelive par exemple est un prestataire avec des serveurs pourris, on a beaucoup d'appels réseaux qui échouent. C'est souvent eux qu'on veut relancer à la main. 
Vous pouvez récupérer ceux qui ont des erreurs auprès de Charline et Elisa